examples/render/contour.yaml

```diff
@@ -209,7 +72,965 @@ data:
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: IngressRouteSpec defines the spec of the CRD
+          properties:
+            routes:
+              description: Routes are the ingress routes. If TCPProxy is present,
+                Routes is ignored.
+              items:
+                description: Route contains the set of routes for a virtual host
+                properties:
+                  delegate:
+                    description: Delegate specifies that this route should be delegated
+                      to another IngressRoute
+                    properties:
+                      name:
+                        description: Name of the IngressRoute
+                        type: string
+                      namespace:
+                        description: Namespace of the IngressRoute. Defaults to the
+                          current namespace if not supplied.
+                        type: string
+                    required:
+                    - name
+                    type: object
+                  enableWebsockets:
+                    description: Enables websocket support for the route
+                    type: boolean
+                  match:
+                    description: Match defines the prefix match
+                    type: string
+                  permitInsecure:
+                    description: Allow this path to respond to insecure requests over
+                      HTTP which are normally not permitted when a `virtualhost.tls`
+                      block is present.
+                    type: boolean
+                  prefixRewrite:
+                    description: Indicates that during forwarding, the matched prefix
+                      (or path) should be swapped with this value
+                    type: string
+                  retryPolicy:
+                    description: The retry policy for this route
+                    properties:
+                      count:
+                        description: NumRetries is maximum allowed number of retries.
+                          If not supplied, the number of retries is zero.
+                        format: int32
+                        type: integer
+                      perTryTimeout:
+                        description: PerTryTimeout specifies the timeout per retry
+                          attempt. Ignored if NumRetries is not supplied.
+                        type: string
+                    required:
+                    - count
```

Comment:
I think count should be +optional based on its comment and the code which will default to 1 if not supplied.

Reply:
Okay, I've updated the comment to that effect as well.